app/code/Magento/ProductVideo/Test/Mftf/ActionGroup/AdminAssertVideoValidationErrorActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertVideoValidationErrorActionGroup">
```

Comment:
Please use Assert[Area]Desctiption naming format for action groups (https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html 11.3.5)